app/assets/css/main.styl

```diff
@@ -1,5 +1,8 @@
 @require 'variables'
 
+*
```

Comment:
That's exactly what I want. The other way is counterintuitive. Imma even expand it per CSS-Tricks recs: https://css-tricks.com/international-box-sizing-awareness-day/